nano/core_test/active_transactions.cpp

```diff
@@ -120,67 +120,71 @@ TEST (active_transactions, confirm_frontier)
+	nano::node_config node_config{ nano::get_available_port (), system.logging };
 	node_config.enable_voting = false;
-	node_config.active_elections_size = 2; //bound to 2, wont drop wallet created transactions, but good to test dropping remote
+	// Bound to 2, won't drop wallet created transactions, but good to test dropping remote
+	node_config.active_elections_size = 2;
 	// Disable frontier confirmation to allow the test to finish before
 	node_config.frontiers_confirmation = nano::frontiers_confirmation_mode::disabled;
+
 	auto & node = *system.add_node (node_config);
 	auto & wallet (*system.wallet (0));
-	//key 1/2 will be managed by the wallet
-	nano::keypair key1, key2, key3, key4, key5, key6;
+
+	nano::keypair key1{};
+	nano::keypair key2{};
+	nano::keypair key3{};
+	nano::keypair key4{};
+	nano::keypair key5{};
+	nano::keypair key6{};
+
 	wallet.insert_adhoc (nano::dev::genesis_key.prv);
-	auto send1 (wallet.send_action (nano::dev::genesis_key.pub, key1.pub, node.config.receive_minimum.number ()));
-	auto send2 (wallet.send_action (nano::dev::genesis_key.pub, key2.pub, node.config.receive_minimum.number ()));
-	auto send3 (wallet.send_action (nano::dev::genesis_key.pub, key3.pub, node.config.receive_minimum.number ()));
-	auto send4 (wallet.send_action (nano::dev::genesis_key.pub, key4.pub, node.config.receive_minimum.number ()));
-	auto send5 (wallet.send_action (nano::dev::genesis_key.pub, key5.pub, node.config.receive_minimum.number ()));
-	auto send6 (wallet.send_action (nano::dev::genesis_key.pub, key6.pub, node.config.receive_minimum.number ()));
-	// should not drop wallet created transactions
+	auto const send1 = wallet.send_action (nano::dev::genesis_key.pub, key1.pub, node.config.receive_minimum.number ());
+	auto const send2 = wallet.send_action (nano::dev::genesis_key.pub, key2.pub, node.config.receive_minimum.number ());
+	auto const send3 = wallet.send_action (nano::dev::genesis_key.pub, key3.pub, node.config.receive_minimum.number ());
+	auto const send4 = wallet.send_action (nano::dev::genesis_key.pub, key4.pub, node.config.receive_minimum.number ());
+	auto const send5 = wallet.send_action (nano::dev::genesis_key.pub, key5.pub, node.config.receive_minimum.number ());
+	auto const send6 = wallet.send_action (nano::dev::genesis_key.pub, key6.pub, node.config.receive_minimum.number ());
+
+	// force-confirm elections one after another
 	ASSERT_TIMELY (5s, node.active.size () == 1);
 	for (auto const & block : { send1, send2, send3, send4, send5, send6 })
 	{
-		ASSERT_TIMELY (1s, node.active.election (block->qualified_root ()));
-		auto election = node.active.election (block->qualified_root ());
-		ASSERT_NE (nullptr, election);
+		std::shared_ptr<nano::election> election{};
+		ASSERT_TIMELY (5s, (election = node.active.election (block->qualified_root ())) != nullptr);
 		election->force_confirm ();
 	}
 	ASSERT_TIMELY (5s, node.active.empty ());
-	nano::state_block_builder builder;
+
+	nano::open_block_builder builder{};
```

Comment:
We're trying to move tests to state blocks only. Is there a reason to change this?